gatsby-node.js

```diff
@@ -55,6 +55,10 @@ exports.onPostBuild = async function(
 
     await Promise.all(chunkJobs);
 
+    if (settings) {
+      tmpIndex.setSettings(settings);
```

Comment:
```suggestion
      indexToUse.setSettings(settings);
```

This way it should also work if there's no index yet. I'll add some tests to this